app/views/handlers/feedback.py

```diff
@@ -224,19 +233,106 @@ def __call__(self) -> Mapping:
 
 
 class FeedbackPayload:
+    """
+    Create the feedback payload object for down stream processing in the following format:
+    ```
+    {
+        "collection": {
+            "exercise_sid": "eedbdf46-adac-49f7-b4c3-2251807381c3",
+            "schema_name": "carbon_0007",
+            "period": "3003"
+        },
+        "data": {
+                "feedback_text": "Page design feedback",
+                "feedback_type": "Page design and structure",
+                "feedback_count": 7,
+        },
+        "metadata": {
+            "ref_period_end_date": "2021-03-29",
+            "ref_period_start_date": "2021-03-01",
+            "ru_ref": "11110000022H",
+            "user_id": "d98d78eb-d23a-494d-b67c-e770399de383"
+        },
+        "origin": "uk.gov.ons.edc.eq",
+        "submitted_at": "2021-10-12T10:41:23+00:00",
+        "started_at": "2021-10-12T10:41:23+00:00",
+        "case_id": "'c39e1246-debd-473a-894f-85c8397ba5ea'",
+        "flushed": False,
+        "survey_id": "001",
+        "form_type: "0007",
+        "tx_id": "5d4e1a37-ed21-440a-8c4d-3054a124a104",
+        "type": "uk.gov.ons.edc.eq:feedback",
+        "launch_language_code: "en",
+        "submission_language_code: "en",
+        "version": "0.0.1"
+    }
+    ```
+    :param metadata: Questionnaire metadata
+    :param started_at: Datetime of questionnaire start
+    :param submitted_at: Datetime of questionnaire submission
+    :param case_id: Questionnaire case id
+    :param schema: QuestionnaireSchema class with populated schema json
+    :param feedback_count: Number of feedback submissions attempted by the user
+    :param feedback_text: Feedback text input by the user
+    :param feedback_type: Type of feedback selected by the user
+    :param feedback_type_question_category: Feedback question category selected by the user
+
+
+    :return payload: Feedback payload object
+    """
+
     def __init__(
         self,
-        feedback_text,
-        feedback_type,
-        feedback_type_question_category=None,
+        metadata: Mapping[str, Union[str, int, list]],
+        started_at: str,
```

Comment:
I know we are conforming to 0.0.1 but seems odd to include the questionnaire started_at in the feedback payload. Even some other keys just seems odd as part of feedback but guess it is what it is.